code/modules/admin/topic.dm

```diff
@@ -347,7 +347,8 @@
 		for(var/jobPos in SSjobs.titles_by_department(COM))
 			if(!jobPos)	continue
 			var/datum/job/job = SSjobs.get_by_title(jobPos)
-			if(!job) continue
+			if(!job || (jobPos in job.alt_titles))
+				continue
```

Comment:
Instead of copypasting the edit for alt-titles everywhere, why don't we edit the jobs subsystem itself? 

I'm thinking we could add a new var on the subsystem to track CORE job titles for each department and ignore the alts. This would also provide a simple way of querying if something is in fact an alt-title - just query `SSjobs.base_titles_by_department()` (or whatever we call it) and if it's not there, it's either an alt/adminbus or something broke _horribly_.

We could also just straight up remove adding the alts to this list (here: https://github.com/Baystation12/Baystation12/blob/f1520a1a0ccf7ff9c1096bcda45f728130836950/code/controllers/subsystems/jobs.dm#L101), but this may have weird side-effects depending on where this method is used and how, so I wouldn't do that. 👀

Reply:
I agree that making a new method within the job system to return only base jobs would be ideal. However, as this was just a quick bugfix PR, I was a little concerned about scope and future maintenance. I also wouldn't touch removing adding alts to that list, lord knows what bay in their infinite wisdom hook into that..

Could add that though if you'd like? Wouldn't take too much effort I'd imagine

Reply:
Hmm, honestly yeah, the odds of us touching that are low vs Bay messing with it. Developing downstream is hell 😿